plugin/src/main/java/com/playmonumenta/scriptedquests/races/RaceFactory.java

```diff
@@ -83,6 +84,16 @@ public RaceFactory(JsonObject object, Plugin plugin, RaceManager manager) throws
 		}
 		mShowStats = show_stats.getAsBoolean();
 
+		// ringless
+		JsonElement ringless = object.get("ringless");
+		if (ringless == null) {
+			//throw new Exception("'ringless' entry is required");
+			//ringless entry is not present at all in a lot of races.
```

Comment:
Pretty reasonable to default to ringless=false, I don't think the commented exception is needed here